deploy/kubernetes/overlays/windows/base/allowedHostPaths.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: csi-gce-pd-node-psp
+spec:
+  allowedHostPaths:
+  - pathPrefix: \var\lib\kubelet
```

Comment:
I think this is going to replace the current PSP with these rules, instead of add, which is not desirable if we're going to be running both linux + windows.

Reply:
good point, will try to make a merge here.

Reply:
actually, I think it is better to have separate policy for linux and windows instead of merging